Medium/SevenBoom.js

```diff
@@ -0,0 +1,18 @@
+// Create a function that takes an array of numbers and return "Boom!" if the digit 7 appears in the array.
+//Otherwise, return "there is no 7 in the array".
+
+const sevenBoom=(arr)=>
+{
+    let temp;
+    for (let i = 0; i < arr.length; i++) {
+        
+        if (arr[i]==7) 
```

Comment:
You can use Array `includes()` method
https://www.w3schools.com/jsref/jsref_includes_array.asp

Reply:
update 
commit ID afe1b612d0d1d14c7e8e6e006203db2b11253cea